vignettes/wip/dm-db.Rmd

```diff
@@ -67,26 +85,42 @@ my_manual_dm <- dm(loans, accounts)
 my_manual_dm
 ``````
 
-- Primary and foreign key constraints on the database interpreted currently only for Postgres and SQL Server databases, need to define manually for other databases
-    - Borrow from blog post
+## Define Primary and Foreign Keys
+
+Primary and Foreign Keys are how relational database tables are linked with each other.
+A primary key is a column (or sometimes a combination of columns) that has a unique value for each row within a table.
+A foreign key is a column (or sometimes a combination of columns) containing the primary key for a row in another table.
```

Comment:
We're working on support for compound keys, they should get a separate short vignette later. Maybe a footnote?

```suggestion
A primary key is a column that has a unique value for each row within a table.
A foreign key is a column containing the primary key for a row in another table.[^compound]

[^compound]: Support for compound keys (consisting of multiple columns) is [planned](https://github.com/krlmlr/dm/issues/3).
```